### PR TITLE
Adds post-processors to fuel column analysis

### LINKED ISSUE #57: Add fuel-column post-processors of the results from Moltres
The repo counts only with the post-processors for obtaining the figures for the fuel-column results from Serpent.
This issue can be closed when the post-processors for obtaining the figures of the results from Moltres is added to the repo.

### PR DESCRIPTION
**Pull request by robfairh**
fixes #57

(I still have issue #56 in mind, I will address it once I have PRed a bunch of files I have around)

### CODE REVIEW

**Review by samgdotson — CHANGES_REQUESTED** [review]
Same story, all functions need return statements. Try not to repeat yourself.

File: serpent-moltres/fuel-column/README.md [hunk 1]
```diff
@@ -1,6 +1,54 @@
+# fuel-column
+
+This section compares Serpent and Moltres results of a fuel column.
+The file `postprocessing.py` produced figures of Serpent and Moltres results.
+
+Serpent input files:
+--------------------
+* standard-column-LBP: column of standard fuel assemblies with bottom and top reflectors. It includes lumped burnable poisons in the corners of the hexagon.
```
Comment (katyhuff) [inline serpent-moltres/fuel-column/README.md:8]: Best practice limits line lengths to 80 chars. Please make this a setting in your text editor. 
```suggestion
* standard-column-LBP: column of standard fuel assemblies with bottom and top reflectors. 
It includes lumped burnable poisons in the corners of the hexagon.
```

Comment (katyhuff) [reply serpent-moltres/fuel-column/README.md:8]: (I won't make comments everywhere else this happens. Please hardwrap lines through this whole PR.)

File: serpent-moltres/fuel-column/postprocessing.py [hunk 2]
```diff
@@ -9,7 +10,198 @@
 from auxiliary import plot_serpent_axial_collapse
 
 
-if __name__ == "__main__":
+def plotcsv_groups_collapse(file, save, lim, dire='z'):
```
Comment (katyhuff) [inline serpent-moltres/fuel-column/postprocessing.py:13]: This postprocessing.py file is a problem. Reused functions should exist in one and only one place in the repo. No files should be copied, pasted, modified, and replicated across multiple files in a repo. Please review the basic reading in this group Best Practices in Scientific Computing.  https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745

Comment (katyhuff) [reply serpent-moltres/fuel-column/postprocessing.py:13]: (one reason this is bad practice is that here I sit, re-reviewing a file that is *almost* identical to one I have reviewed already twice in a previous PR.)

**Review by katyhuff — CHANGES_REQUESTED** [review]
PEP8 violations, DRY violations, copy past of entire postprocessing file... . Please reread https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745

I recognize that since this is MOOSE related, you're probably following the MOOSE style guide which ignores PEP8. That's fine, but it is a bit annoying to violate PEP8 for all python files even the ones not actually having anything to do with moose... like the py files that generate .geo files... I strongly prefer PEP8 unless this is going to end up in the MOOSE test harness.

**Pull request closed without merging**